packages/fxa-payments-server/src/lib/formats.ts

```diff
@@ -1,11 +1,108 @@
-import dayjs from 'dayjs';
+import { FluentNumber, FluentDateTime } from '@fluent/bundle';
 
-export function formatCurrencyInCents(amount: number): string {
-  const decimal = (amount / 100.0).toFixed(2);
-  return `$${decimal}`;
+/**
+ * The following function are for Currency localization
+ */
+
+const baseCurrencyOptions = {
+  style: 'currency',
+  currencyDisplay: 'symbol',
+};
+
+/**
+ * This method is used to provide Fluent with a localizable value that can be formatted per .ftl file based on localization requirements
+ *
+ * @param amountInCents
+ * @param currency
+ */
+export function getLocalizedCurrency(
+  amountInCents: number,
+  currency: string
+): FluentNumber {
+  const decimal = amountInCents / 100;
+  const options = { ...baseCurrencyOptions, currency };
+
+  return new FluentNumber(decimal, options);
+}
+
+/**
+ * This method is to get a default localized currency value for displaying in fallback strings in the event of a Fluent failure.
+ *
+ * @param amountInCents
+ * @param currency
+ */
+export function getLocalizedCurrencyString(
+  amountInCents: number,
+  currency: string
+): string {
+  const decimal = amountInCents / 100;
+  const options = { ...baseCurrencyOptions, currency };
+
+  return new Intl.NumberFormat('en-US', options).format(decimal);
```

Comment:
Is `en-US` suppose to be hard coded here?

Reply:
The logic here: this is the default string that would only be displayed if Fluent fails to translate, currently all of our default strings are en-US. If we use the user's local here (which we totally can), we would run into an issue where the fallback is a mixed language/format. The user would be seeing all of our other default strings (en-US) with a currency that is formatted according to a different local's standard, which could cause confusion.